tests/test_routes.py

```diff
@@ -0,0 +1,54 @@
+def test_get_list_of_planets_with_no_records(client):
+    # Act
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    # Assert
+    assert response.status_code == 200
+    assert response_body == []
+
+def test_get_response_body_of_fixture_planet(client, two_saved_planets):
+    #Act 
+    response = client.get("/planets/1")
```

Comment:
We could made this test a _little_ less brittle, though perhaps a little tougher to read, if we write `two_saved_planets` to return the created models as well. We could then access the id of the model here, using the id the database gave the record. This can help us future-proof our tests (in case we change the number or order of records created, directly or indirectly).